### DISCUSSION TITLE
设置文件路径在哪

### DISCUSSION THREAD

**Discussion by superen7527** [the1812/Bilibili-Evolved#3439] [提问 / Q&A]
好几天用不了，开脚本就是黑屏白屏。开梯子没用。用了预览版本可以，正式版进不去导出设置 故此问手动导出

**Comment by the1812** [comment]
![image](https://user-images.githubusercontent.com/26504152/174470613-c7c91b7c-8aad-49ef-923b-df4dcc24c3da.png)
编辑脚本, 选择 `存储`, 里面的东西复制出来就是

**Reply by superen7527** [reply]
> ![image](https://user-images.githubusercontent.com/26504152/174470613-c7c91b7c-8aad-49ef-923b-df4dcc24c3da.png) 编辑脚本, 选择 `存储`, 里面的东西复制出来就是

我的是暴力猴 没有这个。。。有本地路径吗没找到

**Reply by the1812** [reply]
编辑 数据 全部, 右边就是
![image](https://user-images.githubusercontent.com/26504152/177169757-d72b88e6-f251-4d44-9838-e2dba6ebf8ec.png)
